.changesets/fix_garypen_http_ext_clone_works_properly.md

```diff
@@ -0,0 +1,9 @@
+### Take advantage of improvement in the http crate for cloning ([PR #7559](https://github.com/apollographql/router/pull/7559))
```

Comment:
I think this changeset contains too much technical depth for a typical user. Maybe we could focus just on the known case that was fixed here?